lib/ansible/module_utils/oneview.py

```diff
@@ -202,7 +202,12 @@ class OneViewModuleBase(object):
     HPE_ONEVIEW_SDK_REQUIRED = 'HPE OneView Python SDK is required for this module.'
 
     ONEVIEW_COMMON_ARGS = dict(
-        config=dict(required=False, type='str')
+        config=dict(required=False, type='path'),
+        hostname=dict(required=False, type='str'),
```

Comment:
@dagwieers Is this the expected/wanted solution to not use a `json` file?
The task ended up looking like:
```yaml
    - name: Gather facts about all Network Sets
      oneview_network_set_facts:
        hostname: 172.16.101.48
        username: administrator
        password: my_password
        api_version: 500
      no_log: true
      delegate_to: localhost
```

I also had to use no_log otherwise the password was being logged and outputted to the screen when using `-vvv` during playbook execution.

Reply:
Correct ! no_log is needed for password parameter

Reply:
Again `required=False` is not needed here. Only add it when it is required.